specification/azsadmin/resource-manager/user-subscriptions/readme.md

```diff
@@ -35,6 +35,9 @@ directive:
     where:
       - $.paths["/subscriptions/{subscriptionId}"].put
 
+  - suppress: R3023
+    reason: No operations endpoint as not ARM resource provider.
```

Comment:
What's the reason for this suppression? Should we have ARM approval for this? Is it for a specific place? In this case, should we use `where` and `from`.

Reply:
There is no operations endpoint in ARM.  This is almost the same as the Microsoft.Subscription spec but we have different objects.

Reply:
This suppression is OK from ARM's perspective.